code/game/objects/items/ego_weapons/non_abnormality/weak_edits/city.dm

```diff
@@ -111,7 +111,7 @@
 		TEMPERANCE_ATTRIBUTE = 60,
 		JUSTICE_ATTRIBUTE = 60,
 	)
-	multiplier = 3
+	multiplier = 5
```

Comment:
bruh
Please change it back these fuckers are strong

Reply:
Honestly I more changed it to 5 because this is still technically a nerf since theyre going from the 48 damage blade to the 30 but if you want the requested buff reverted sure